### DISCUSSION TITLE
docker手动升级时环境变量会丢失

### DISCUSSION THREAD

**Discussion by SoulGolf** [1Panel-dev/1Panel#5626] [Q&A]
### 1Panel 版本

v1.10.11-lts

### 问题描述

docker容器手动升级镜像版本时环境变量会丢失,例如 one-api,升级成功后环境变量恢复了默认配置

### 重现步骤

docker容器手动升级镜像版本时

**Comment by zhengkunwang223** [comment]
不太建议使用容器手动升级，尤其是从应用商店安装的应用
<img width="675" alt="image" src="https://github.com/1Panel-dev/1Panel/assets/31820853/010d94c9-d5e7-4f63-8a4e-a4920c39f549">

应用商店安装的应用环境变量维护在安装目录的 .env 文件中
